CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Enhanced YAML test runner to use OpenSearch rest-api-spec YAML tests ([#414](https://github.com/opensearch-project/opensearch-py/pull/414)
 - Added `Search#collapse` ([#409](https://github.com/opensearch-project/opensearch-py/issues/409))
 - Added support for the ISM API ([#398](https://github.com/opensearch-project/opensearch-py/pull/398))
+- Added `trust_env` parameter to `AIOHttpConnection` class ([#398](https://github.com/opensearch-project/opensearch-py/pull/438))
```

Comment:
Can I nitpick? :) Say "Aded AIOHttpConnection(trust_env)` maybe or just "Added `trust_env` to `AIOHttpConnection`", I think parameter and class are redundant ;)